hstream/src/HStream/Server/Handler.hs

```diff
@@ -285,11 +285,15 @@ handleDropPlan :: ServerContext -> Bool -> DropObject
   -> IO (ServerResponse 'Normal CommandQueryResponse)
 handleDropPlan sc@ServerContext{..} checkIfExist dropObject = defaultExceptionHandle $ do
   case dropObject of
-    DStream stream -> handleDrop stream transToStreamName
-    DView view     -> do
+    DConnector connector -> handleDrop connector transToStreamName
```

Comment:
I am afraid that it can not work as you expected. Take a test to see if your implementation can work. I mean, create a connector and try `DROP CONNECTOR` to find out if you can actually drop it. If it is inconvenient to run the code, you can read the code to see how a connector is created, and then you can get more information about how to drop it.

Reply:
And maybe the name `handleDrop` is somehow misleading. It will be rewritten soon.